cmd/localstack/main.go

```diff
@@ -60,6 +62,19 @@ func main() {
 	log.SetLevel(log.DebugLevel)
 	log.SetReportCaller(true)
 
+	// Switch to non-root user and drop root privileges
+	if IsRootUser() && lsOpts.User != "" {
+		uid := 993
+		gid := 990
+		AddUser(lsOpts.User, uid, gid)
+		if err := os.Chown("/tmp", uid, gid); err != nil {
+			log.Errorln("Error changing owner of /tmp:", err)
+		}
+		UserLogger().Debugln("Process running as root user.")
+		DropPrivileges(lsOpts.User)
+		UserLogger().Debugln("Process running as non-root user.")
+	}
+
```

Comment:
Not a fan of doing this before we start the DNS. Depending on a docker version (https://github.com/moby/moby/pull/41030), we might not have the permission for binding a port < 1024 later. I would move the privilege drop after the code archive extraction + dns server start.

Reply:
Makes a lot of sense. Good catch 👍 
Moved it right after `go RunDNSRewriter`